sources/packages/api/src-sql/Student/Add-SINValidation.sql

```diff
@@ -0,0 +1,6 @@
+ALTER TABLE
+    sims.students
+ADD
+    COLUMN IF NOT EXISTS sin_validations_id INT REFERENCES sims.sin_validations(id);
```

Comment:
I believe that we have been using the singular for relationships: _sin_validation_id_